utils/removeDiscordRoleFromUser.js

```diff
@@ -0,0 +1,52 @@
+const discordRolesModel = require("../models/discordactions");
+const discordServices = require("../services/discordService");
+const { logType } = require("../constants/logs");
+const { addLog } = require("../models/logs");
+
+/**
+ * Removes a Discord role from a user using Discord Id.
+ *
+ * @param {Object} userData - User data.
+ * @param {string} discordId - User's Discord ID.
+ * @param {string} roleId - Discord Role ID.
+ *
+ * @returns {Promise<Object>} - Result with success status and message.
+ */
+export const removeDiscordRoleFromUser = async (userData, discordId, roleId) => {
+  try {
+    const { roleExists } = await discordRolesModel.isGroupRoleExists({ roleid: roleId });
+
+    if (!roleExists) {
+      const message = "Role doesn't exist";
+      await addLog(logType.REMOVE_ROLE_FROM_USER_FAILED, { roleId }, { message, userid: discordId, userData });
```

Comment:
Are we logging errors in db for others api?

Reply:
Maybe no. But I think it is a good practice to store log errors in db for mutation queries.